Replace Exploration Statistics Tab with Additional Player Features in GitHub Actions E2E

U8NWXD opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of #n/a.
2. This PR does the following: The explorationStatisticsTab suite in GitHub Actions is flaking, so we replace it with the additionalPlayerFeatures suite. We also remove the unnecessary `DISPLAY` environment variable (see the commit message for 8dbd3aa for why)

## Essential Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## PR Pointers

- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays
- Never force push. If you do, your PR will be closed.

## Run History

1. [All pass](https://github.com/oppia/oppia/pull/10489/checks?check_run_id=1051869559)

![Screen Shot 2020-08-31 at 12 05 39](https://user-images.githubusercontent.com/19878639/91741166-50acd000-eb82-11ea-857c-c4ec3fa6765e.png)

2. [All pass](https://github.com/oppia/oppia/pull/10489/checks?check_run_id=1052040490)

![Screen Shot 2020-08-31 at 12 48 28](https://user-images.githubusercontent.com/19878639/91744993-4ee60b00-eb88-11ea-9948-637d317abaf6.png)

3. [All pass](https://github.com/oppia/oppia/pull/10489/checks?check_run_id=1052205570)

![Screen Shot 2020-08-31 at 13 23 06](https://user-images.githubusercontent.com/19878639/91747962-26144480-eb8d-11ea-9475-c2cfb11780f3.png)

U8NWXD committed:
Remove DISPLAY Env Variable from E2E Actions

The DISPLAY environment variable is not needed when we use xvfb-run -a,
like we do in GitHub Actions for the e2e tests, because -a causes xvfb
to find an open display, so we don't need to specify one.

U8NWXD committed:
Move Additional Player Features to GitHub Actions

The explorationStatisticsTab suite is flaking, so we replace it with
additionalPlayerFeatures.

BenHenning reviewed (commented):
Thanks @U8NWXD. One thing that I'm confused about: how is additionalPlayerFeatures a substitute for explorationStatisticsTab? Shouldn't we be running both test suites?

U8NWXD commented:
@BenHenning Both suites currently run on TravisCI. GitHub Actions currently duplicates some TravisCI tests. Once we are confident these tests run will in GitHub Actions, we will remove them from TravisCI. When that happens, we will remove additionalPlayerFeatures, but not explorationStatisticsTab

U8NWXD committed:
Merge branch 'develop' into swap-github-actions-e2e

BenHenning commented:
Sounds good, thanks @U8NWXD.

U8NWXD committed:
Replace contributorDashboard with accessibility

Contributor dashboard suite was flaking.

U8NWXD committed:
Swap additionalPlayerFeatures for classroomPage

additionalPlayerFeatures flaked (e.g.
https://github.com/oppia/oppia/pull/10489/checks?check_run_id=1049061919),
so we don't want it on GitHub Actions. classroomPage has never flaked on
develop.
